specification/src/chapter5.adoc

```diff
@@ -2,6 +2,66 @@
 
 == Cryptography
 
-<use models enabled by crypto ISA extensions>
+RISC-V supports a number of ISA-level extensions aimed at improving performance and reducing the risk of side-channel attacks for cryptographic operations (scalar and vector). They also include an ISA-level entropy source, and guidelines for data independent execution latency.
```

Comment:
imo the crypto extensions are mainly for performance -  they dont really help much with side channels as they are single round based. the HA extension primarily addresses this, and adds per privilege key management options.

Reply:
Depends on the side channel. The instructions defend against cache timing side channels caused by T-table based AES implementations. They may not defend against differential power analysis based side channels.

Reply:
refer to DIEL Zkt, Zkvt

Reply:
Updated the text.